README.md

```diff
@@ -89,6 +89,20 @@ This config option controls the export file format for each Sphinx builder. It
 accepts a dictionary mapping builder names to image formats. Accepted values for
 the latter are `"png"`, `"jpg"`, or `"svg"`.
 
+### Output directory
+- *Formal Name*: `drawio_export_directory`
+- *Default Value*: `None`
+
+This config option controls where the generated files are exported to.
+Both relative (to the current working directory) and absolute paths
+are supported.
+
+This can be useful to reduce build times with many drawio files across
+multiple sphinx build formats.
+
+If not set all files are generated into the builders output directory
+e.g. `_build/html/drawio/*`
```

Comment:
We should probably mention that the exported images in this directory are not being referenced by the built documentation and that it can safely be removed.